### DISCUSSION TITLE
How does ExtPay take the 5% cut?

### DISCUSSION THREAD

**Discussion by anonymous121212** [Glench/ExtPay#230] [Q&A]
I am new here. Before deciding to use ExtPay, I would like to understand how it works. (1) How does extpay know how much revenue generated? (2) How extpay takes the 5% cut? Stripe does the split to pay extpay the 5% and the remaining to developers? Or we need to pay the 5% on a regular basis? Thanks

**Comment by burkybang** [comment]
It's all automatic through Stripe.

**Comment by Glench** [comment]
The payments are made through Stripe and Stripe has a method for taking a "platform fee" at the time of purchase, which is what ExtPay uses for most accounts. For accounts in India, Mexico, Brazil, and a few other countries, Stripe doesn't allow platform fees so we just invoice you monthly for sales your extension makes.
